pom.xml

```diff
@@ -596,6 +601,11 @@
                 <version>1.7.10</version>
             </dependency>
             <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-log4j12</artifactId>
+                <version>1.7.25</version>
```

Comment:
Let's move to the latest version of log4j as well.  We don't want to fall too far behind.